.slather.yml

```diff
@@ -0,0 +1,7 @@
+coverage_service: cobertura_xml
+xcodeproj: platforms/ios/example/Wysiwyg.xcodeproj
+source_directory: platforms/ios/example/lib
```

Comment:
```suggestion
source_directory: platforms/ios/lib
```

Reply:
(not sure what implies each of these directories, but that one is not a valid address :) )